src/main.py

```diff
@@ -151,7 +200,11 @@ def matricule(frame, outs, width_factor=1.1, height_factor=1.0):
     except Exception as e:

```

Comment:
In line 184 instead of unnecessary write/read operation , try passing it directly from the detected_plates , process it in memory and extract the "plate_temp" and pass it to input_plate.